packages/framework/client-logger/app-insights-logger/tsconfig.json

```diff
@@ -4,7 +4,7 @@
 		"rootDir": "src",
 		"outDir": "dist",
 		"composite": true,
-		"types": ["mocha"],
```

Comment:
Was this necessary in the end? Not sure I saw something that needs it.

Reply:
If you remove it my mocha tests will complain about missing types.